modules/dataquery/js/workers/savezip.js

```diff
@@ -1,5 +1,6 @@
 /*global self: false, Blob: false */
-importScripts(Loris.BaseURL + "/js/jszip/jszip.min.js");
+var BaseURL = self.location.href.split(self.location.pathname)[0];
```

Comment:
can't this use loris.BaseURL (like above)? Or does that not work because it's a web worker? I'm not sure that this is going to be reliable if it's not directly served from the web server document root.